notebooks/vllm_inference_engine.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<p>Can you explain briefly what <code><strong>max_model_len</strong></code><strong> </strong>is? Is it the length of the <em>prompt</em> or <em>prompt + response</em> or only <em>response</em>?</p><p>Also, small typos:</p><ul><li>models <strong>are</strong> automatically downloaded from Hugging Face</li><li>...that <strong>allows</strong> LLMs to run (fast)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>